### ISSUE TITLE
Crash on server launch

### ISSUE THREAD

**Issue by Sunekaer** [CyclopsMC/IntegratedDynamics#1095]
#### Issue type:

- :bug: Bug

____

#### Short description:

Crash on server launch

#### Steps to reproduce the problem:

start server

#### Expected behaviour:

to not crash

____

#### Versions:

- This mod: IntegratedDynamics-1.16.5-1.10.0.jar      
- Minecraft: 1.16.5
- Forge: 36.2.2

#### Log file:

https://paste.sunekaer.com/bizohacuyo.rb

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by rubensworks** [comment]
Duplicate of #1093. New release is pending on CurseForge.
